### DISCUSSION TITLE
SVT-Play_saknad text_problemlösning

### DISCUSSION THREAD

**Discussion by 68456gjp49586g** [stefansundin/privatkopiera#223] [General]
ffmpeg-7.0.1 löser inte problemet med saknad text. Dock ger ett mindre antal text
Man kan se att Inställningen i spelaren bestämmer om man får texten eller inte. (se bifogad bild)
När "stora" rutan (med Undertexter svenska av/på) visas kommer inte texten med.
När "lilla" rutan (utan Undertexter svenska av/på)  visas kommer texten med.
Hoppas detta kan ge inspiration till problemlösning.
<img width="548" alt="svtplay_text" src="https://github.com/stefansundin/privatkopiera/assets/77499264/8a43c600-ee35-4589-9f96-3988a692d4c0">

**Comment by webberian** [comment]
Jag tror det du visar exempel på är när det finns en separat undertextfil (då visas "stora rutan" med Undertexter-alternativet) och när undertexten istället är "inbränd" direkt i videofilen (och därmed varken går att slå av eller på, din "lilla ruta").

**Comment by 68456gjp49586g** [comment]
Hej
Ett nytt problem utan valbar text:
https://www.svtplay.se/video/KRo2b9E/walk-the-line?video=visa

Slutar ladda direkt.
Något med nytt ljudformat?
Använder ffmpeg-master-latest-win64-gpl_bin_
Hälsning/G


```
[aac_adtstoasc @ 000001d9973cac00] Codec 'ac3' (86019) is not supported by
the bitstream filter 'aac_adtstoasc'. Supported codecs are: aac (86018)
[aost#0:1/copy @ 000001d9971aed00] Error initializing bitstream filter:
aac_adtstoasc
Error opening output file Walk the Line [SVT eE37X97].mp4.
Error opening output files: Invalid argument
```

**Reply by stefansundin** [reply]
Fixat i v0.4.4.
